docker-compose.debug.yaml

```diff
@@ -0,0 +1,23 @@
+# Adds ports to the db and access to the temporal UI for debugging purposes.
+# Expected to be used like this:
+# VERSION=dev docker-compose -f docker-compose.yaml -f docker-compose.debug.yaml up
+version: "3.7"
+x-logging: &default-logging
+  options:
+    max-size: "100m"
+    max-file: "5"
+  driver: json-file
+services:
+  db:
+    ports:
+      - 5432:5432
```

Comment:
i'm not a big fan of using 5432 in this case. if someone is running postgres on their local machine it is super likely that port is being used by that instance. i am in favor of picking a less commonly used port.

Reply:
or having a comment explaining that you can change the port if you do run into a conflict.